custom_components/miele/translations/de.json

```diff
@@ -294,14 +294,14 @@
                     "white_tea": "Weißer Tee",
                     "japanese_tea": "Japanischer Tee",
                     "green_tea": "Grüner Tee",
-                    "fruit_tea": "Früchtetee",
-                    "herbal_tea": "Kräutertee",
+                    "fruit_tea": "Früchte-Tee",
```

Comment:
TBH, I think I'd prefer the compound word `Früchtetee` here, "Früchte-Tee" looks really jarring.